### ISSUE TITLE
cognito-identity-provider `notify()` method clash

### ISSUE THREAD

**Issue by alexcardell** [disneystreaming/smithy4s#1238]
Example project

```sbt
lazy val lambda = project
  .in(file("lambda"))
  .enablePlugins(Smithy4sCodegenPlugin)
  .settings(
    libraryDependencies ++= Seq(
      "com.disneystreaming.smithy4s" %%% "smithy4s-aws-http4s" % smithy4sVersion.value,
      "com.disneystreaming.smithy" % "aws-cognito-identity-provider-spec" % "2023.09.22" 
    )
  )
```

With no source files of my own, this is the compiler output:

```
/lambda/target/scala-2.13/src_managed/main/scala/com/amazonaws/cognitoidentityprovider/AccountTakeoverActionType.scala:36:83: cannot override final member:
[error] final def notify(): Unit (defined in class Object)
[error] case class AccountTakeoverActionType(eventAction: AccountTakeoverEventActionType, notify: AccountTakeoverActionNotifyType = com.amazonaws.cognitoidentityprovider.AccountTakeoverActionNotifyType(false))
[error]                                                                                   ^
[error] one error found
[error] (lambda / Compile / compileIncremental) Compilation failed
```

This is a ScalaJS project, intended to be used with feral. There are no differences with the LambdaJSPlugin applied.

Also validated with v2023.02.10 of the specs

**Comment by daddykotex** [comment]
Wow, this is new to me. And I'm not sure how I did not know all these methods existed on [Object](https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notify--)

I'll have a look, and circle back quickly

**Comment by daddykotex** [comment]
thanks for the issue. we're on it

**Comment by kubukoz** [comment]
#1231 will have it so I assume it'll be available in 0.18.0.

**Comment by Baccata** [comment]
Indeed. The fix will be rolled in 0.18.0
